code/forms/SiteTreeURLSegmentField.php

```diff
@@ -30,7 +30,7 @@ public function getAttributes() {
 			parent::getAttributes(),
 			array(
 				'data-prefix' => $this->getURLPrefix(),
-				'data-suffix' => '?stage=Stage',
+				'data-suffix' => '?stage=' . $this->getURLSuffix(),
```

Comment:
Huh where? This thread? Sorry I didn't get that from the conversation above... I guess I'm just slow today. :D